compiler/rustc_codegen_llvm/src/va_arg.rs

```diff
@@ -266,7 +266,7 @@ pub(super) fn emit_va_arg<'ll, 'tcx>(
         // Generic x86
         "x86" => emit_ptr_va_arg(bx, addr, target_ty, false, Align::from_bytes(4).unwrap(), true),
         // Windows AArch64
-        "aarch64" if target.is_like_windows => {
+        "aarch64" | "arm64ec" if target.is_like_windows => {
```

Comment:
I would have thought that the alignment would be the same as `x86`? Can you confirm this is the correct alignment to use?

Reply:
ARM64EC va-args has the same alignment as aarch64, and then LLVM will generate the extry/exit "thunks" to convert to/from x64 calling convention and those will have x86 alignment.

Reply:
Ah, ok. Thanks!